src/components/anchor/code/templates/anchorTemplate.jsx

```diff
@@ -0,0 +1,16 @@
+var React = require('react');
+
+module.exports = function() {
+
+  if(!this.props.children) {
+    return null;
+  }
+
+  var dataAttributes = this.getDataAttributesFromProps();
+
+  return (
```

Comment:
@Joezo to be honest, I thought we were going to phase out the idea of including templates like this anyway?